checkov/terraform/checks/data/aws/IAMPublichActionsPolicy.py

```diff
@@ -0,0 +1,32 @@
+from typing import Dict, List, Any
+
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.common.util.type_forcers import force_list
+from checkov.terraform.checks.data.base_check import BaseDataCheck
+
+
+class IAMPublicActionsPolicy(BaseDataCheck):
+    def __init__(self):
+        name = 'Ensure no IAM policies documents allow "*" as a statement\'s actions'
+        id = "CKV_AWS_284"
```

Comment:
we already have a check `CKV_AWS_49`

Reply:
That checks the actions. I want to check the principal - different check @gruebel